tsconfig.test.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "target": "es6"
```

Comment:
`es6` for testing so that I can get access to `URLSearchParams.entries()`, shouldn't cause any problems with the SDK at runtime?